src/vs/editor/standalone/common/monarch/monarchLexer.ts

```diff
@@ -424,6 +426,16 @@ export class MonarchTokenizer implements languages.ITokenizationSupport {
 				emitting = false;
 			}
 		});
+		this._maxTokenizationLineLength = this._configurationService.getValue<number>('editor.maxTokenizationLineLength', {
+			overrideIdentifier: this._languageId
+		});
+		this._configurationService.onDidChangeConfiguration(e => {
```

Comment:
Should this listener be disposed at some point?

Reply:
Good point! I'll do a follow-up PR.